### DISCUSSION TITLE
FEATURE: inconLeft iconRight for <Button>, like chakra ui:

### DISCUSSION THREAD

**Discussion by vitalijalbu** [ant-design/ant-design#47325] [Ideas]
FEATURE: inconLeft iconRight for <Button>, like chakra ui:
[https://chakra-ui.com/docs/components/button/usage](https://chakra-ui.com/docs/components/button/usage)

**Comment by li-jia-nan** [comment]
I think it's a good idea, we can call it the `iconStart` and `iconEnd`.

**Comment by h-bessa** [comment]
Any news about this feature ?

**Comment by vitalijalbu** [comment]
@li-jia-nan 😃

**Comment by h-bessa** [comment]
Little trick

`<Button type="primary"> Button right text <Icon /> </Button>`

**Reply by vitalijalbu** [reply]
😟really?? 🤨

**Comment by afc163** [comment]
- #47791
